package.json

```diff
@@ -110,7 +110,7 @@
     "@krakenjs/zoid": "^10.3.1",
     "@paypal/common-components": "^1.0.35",
     "@paypal/funding-components": "^1.0.31",
-    "@paypal/connect-loader-component": "^1.1.0",
+    "@paypal/connect-loader-component": "1.1.0-beta.1",
```

Comment:
We need to update `package_lock.json` too?

Reply:
@siddy2181 there isn't a lock file in checkout components.
We disable the lockfile generation here: https://github.com/paypal/paypal-checkout-components/blob/main/.npmrc#L3